test/error.js

```diff
@@ -90,10 +90,13 @@ test.serial('http.request error', async t => {
 });
 
 test.serial('catch error in mimicResponse', async t => {
+	const mimicResponse = () => {
+		throw new Error('Error in mimic-response');
+	};
+	mimicResponse['@global'] = true;
```

Comment:
Not best practice to use `@global` with `proxyquire` but it does not _currently_ result in any side-effects in the tests. An alternative would be to introduce a `mimic-response-wrapper.js` that exports a named function we can mock with `Sinon` and then use that in got.

Reply:
Nah, I think what you have here is fine.